Polyhedron/demo/Polyhedron/Plugins/AABB_tree/Cut_plugin.cpp

```diff
@@ -799,7 +799,7 @@ class Polyhedron_demo_cut_plugin :
 
    ~Polyhedron_demo_cut_plugin();
 
-  bool applicable(QAction*) const {
+  bool applicable(QAction*) const Q_DECL_OVERRIDE{
```

Comment:
Next time, remember that once we use C++11, we can use the real `override` keyword.

Reply:
Ah ? `override` is not a Qt keyword, from the ones we disabled ? I didn't know.

Reply:
No, that is a C++11 keyword, from the language itself, like `virtual`:
  https://en.cppreference.com/w/cpp/language/override